File: beginner/javascript/1073.js
var input = require('fs').readFileSync('stdin', 'utf8');

var valores = input.split('\n');

var i, quadrado;

var n = parseInt(valores.shift());


for(i = 1; i <= n; i++){
    if(i % 2 == 0){
        quadrado = Math.pow(i,2);
        console.log(i + "^2 = " + quadrado);
    }
}

File: beginner/c/1073.c
#include <stdio.h>
#include <math.h>

int main(){

    int i, n, quadrado;

    scanf("%d", &n);

    for(i = 1; i <= n; i++){
        if(i % 2 == 0){
            quadrado = pow(i,2);
            printf("%d^2 = %d\n", i, quadrado);
        }
    }

    return 0;
    
}